library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileInputOperator.java

```diff
@@ -384,6 +388,9 @@ public int getEmitBatchSize()
    */
   public void setEmitBatchSize(int emitBatchSize)
   {
+    if (emitBatchSize <= 0) {
+      throw new IllegalArgumentException("emitBatchSize should be greater than 0.");
+    }
```

Comment:
Can we add the validation annotation to emitBatchSize, instead fixing it in setter method ?